src/bech32.cpp

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2017 Pieter Wuille
```

Comment:
Is this the right copyright header (here and in `bech32.h`)?

Reply:
I think so.

Reply:
Ok, just checking since most files in this repo have `Copyright (c) The Bitcoin Core developers`